bundles/org.openhab.binding.kvv/src/main/java/org/openhab/binding/kvv/internal/DepartureResult.java

```diff
@@ -0,0 +1,315 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.kvv.internal;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Represents the result of a call to the KVV api to fetch the next departures at a specific stop.
+ *
+ * @author Maximilian Hess - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class DepartureResult {
+
+    /** timestamp */
+    private String timestamp = "";
+
+    /** name of the stop */
+    private String stopName = "";
+
+    /** Ordered list of departures */
+    private List<Departure> departures = new ArrayList<Departure>();
+
+    /**
+     * Creates a new and empty {@link DepartureResult}
+     */
+    public DepartureResult() {
+    }
+
+    /**
+     * Returns the timestamp.
+     *
+     * @return the timestamp.
+     */
+    public String getTimestamp() {
+        return timestamp;
+    }
+
+    /**
+     * Sets the timestamp.
+     *
+     * @param timestamp the new timestamp.
+     */
+    public void setTimestamp(String timestamp) {
+        this.timestamp = timestamp;
+    }
+
+    /**
+     * Returns the name of the stop.
+     *
+     * @return the name of the stop.
+     */
+    public String getStopName() {
+        return stopName;
+    }
+
+    /**
+     * Sets the name of the stop.
+     *
+     * @param stopName the new name of the stop
+     */
+    public void setStopName(String stopName) {
+        this.stopName = stopName;
+    }
+
+    /**
+     * Returns an ordered list of the next departures.
+     *
+     * @return an ordered list of the next departures.
+     */
+    public List<Departure> getDepartures() {
+        return departures;
+    }
+
+    /**
+     * Sets the ordered list of the next departures.
+     *
+     * @param departures the new departures
+     */
+    public void setDepartures(List<Departure> departures) {
+        this.departures = departures;
+    }
+
+    @Override
+    public String toString() {
+        return "DepartureResult [timestamp=" + timestamp + ", stopName=" + stopName + ", departures=" + departures
+                + "]";
+    }
+
+    /**
+     * Encapsulates a single {@link Departure}. Most important are 'route', 'destination' and 'destination'.
+     *
+     * @author Maximilian Hess - Initial contribution
+     *
+     */
+    @NonNullByDefault
```

Comment:
Inner classes don't need to declare NonNullByDefault. When you wrote the code there was probably still the bug in the checkstyle rules, which generated a warning here.